packages/ckeditor5-engine/src/model/writer.ts

```diff
@@ -745,31 +811,29 @@ export default class Writer {
 	/**
 	 * Shortcut for {@link module:engine/model/model~Model#createRangeOn `Model#createRangeOn()`}.
 	 *
-	 * @param {module:engine/model/element~Element} element Element which is a parent for the range.
-	 * @returns {module:engine/model/range~Range}
+	 * @param element Element which is a parent for the range.
 	 */
 	public createRangeOn( element: Item ): Range {
 		return this.model.createRangeOn( element );
 	}
 
+	// The three overloads below where added,
+	// because they render better in API Docs than rest parameter with union of tuples type (see the constructor of `Selection`).
+	public createSelection(): Selection;
+	public createSelection( selectable: Selectable, placeOrOffset?: PlaceOrOffset, options?: { backward?: boolean } ): Selection;
+	public createSelection( selectable: Selectable, options: { backward?: boolean } ): Selection;
+
 	/**
 	 * Shortcut for {@link module:engine/model/model~Model#createSelection `Model#createSelection()`}.
-	 *
-	 * @param {module:engine/model/selection~Selectable} selectable
-	 * @param {Number|'before'|'end'|'after'|'on'|'in'} [placeOrOffset] Sets place or offset of the selection.
-	 * @param {Object} [options]
-	 * @param {Boolean} [options.backward] Sets this selection instance to be backward.
-	 * @returns {module:engine/model/selection~Selection}
 	 */
-	public createSelection( ...args: ConstructorParameters<typeof Selection> ): Selection {
+	public createSelection( ...args: [ any?, any?, any? ] ): Selection {
```

Comment:
I'm not sure if this would look nice in API docs.

Reply:
It's not in API docs. This is an implementation signature and exists because it is difficult to forward overloaded method parameters to another overloaded method.